### PR TITLE
Fix clientIO stressbench throughput calculation

### PR DESCRIPTION
**Pull request by ssz1997**
Divide 1000 to get megabytes per second instead of megabytes per millisecond.

### CODE REVIEW

File: stress/common/src/main/java/alluxio/stress/client/ClientIOTaskResult.java [hunk 1]
```diff
@@ -262,7 +262,7 @@ public ClientIOSummary aggregate(Iterable<ClientIOTaskResult> results) throws Ex
       for (Map.Entry<Integer, Long> threadCountIOBytesEntry: threadCountIOBytes.entrySet()) {
         int numThreads = threadCountIOBytesEntry.getKey();
         long ioBytes = threadCountIOBytesEntry.getValue();
-        threadCountIoMbps.put(numThreads, (float) ioBytes
+        threadCountIoMbps.put(numThreads, (float) ioBytes / 1000
```
Comment (yyongycy) [inline stress/common/src/main/java/alluxio/stress/client/ClientIOTaskResult.java:265]: are we usually using 1000 or 1024? any constants?

Comment (dbw9580) [reply stress/common/src/main/java/alluxio/stress/client/ClientIOTaskResult.java:265]: I think for `MBps` you should be doing something like this:

```java
public float getIOMBps() {
  return ((float) mIOBytes / getDurationMs()) * 1000.0f / Constants.MB;
}
```

Comment (ssz1997) [reply stress/common/src/main/java/alluxio/stress/client/ClientIOTaskResult.java:265]: Thanks. I messed up the math

**Review by yyongycy — APPROVED** [review]
lgtm

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**